src/Components/Admins/Form/form.module.css

```diff
@@ -46,6 +38,8 @@
   display: flex;
   flex-direction: column;
   margin-bottom: 0.5rem;
```

Comment:
Based on good practices and conventions, the content related to flex should come before the content related to box model. So margin-bottom should be after justify-content and align-items